rust/https_outcalls_GET_request/src/hello_http_rust_backend/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+//1. DECLARE IC MANAGEMENT CANISTER
+//This also includes methods and types needed
+use ic_cdk::api::management_canister::http_request::{
+    http_request, CanisterHttpRequestArgument, HttpHeader, HttpMethod, HttpResponse, TransformArgs,
+    TransformContext,
+};
+
+//Update method using the HTTPS outcalls feature
+#[ic_cdk::update]
+async fn get_icp_usd_exchange() -> String {
+
+  //2. SETUP ARGUMENTS FOR HTTP GET request
+
+    // 2.1 Setup the URL and its query parameters
+    type Timestamp = u64;
+    let start_timestamp : Timestamp = 1682978460; //May 1, 2023 22:01:00 GMT
```

Comment:
This code in general should be run through `cargo fmt`.

Reply:
done